keps/sig-apps/2307-job-tracking-wihout-lingering-pods/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Job tracking without lingering Pods
+kep-number: 2307
+authors:
+  - "@alculquicondor"
+owning-sig: sig-apps
+status: implementable
+creation-date: 2020-01-21
+reviewers:
+  - "@erictune"
+  - "@lavalamp"
+  - "@soltysh"
+approvers:
+  - "@janetkuo"
+prr-approvers:
+- "@wojtek-t"
```

Comment:
Please change to me, Wojtek is over scheduled.

Reply:
I triggerred that discussion, so I will continue looking into it.